### DISCUSSION TITLE
APSIM requires admin access to run?

### DISCUSSION THREAD

**Discussion by edvlmaas** [APSIMInitiative/ApsimX#8910] [Q&A] [answered]
Hi, my organization does not allow users admin rights to their computers, so I had IT install APSIM 2024.5.7471.0, which they put on a D: drive. I have a colleague who has APSIM (2023.9.7309.0) installed and configured this way and it works for him. However, at the moment, only the IT person can launch my program by right-clicking and selecting "Run as administrator". I cannot start it. All the permissions on the .exe look in order to allow non-admin people to run the program. Has anyone else run into this?

**Comment by lie112** [comment]
This sounds like a file permissions issue with the directory where APSIM is installed. What if you copy the APSIM files installed on d:\ to a public directory or on your c: drive (e.g. C:\Temp\APSIM\) and also ensure the directory where the simulation file is stored is also a public folder. (e.g. c:\Temp\APSIM\simulations\)? APSIM will need to write to a config file and also update your simulation file and so won't need admin privileges if working in a public location. This is what you might want to check first.

**Comment by hol353** [comment, marked as answer]
At install time, APSIM asks whether you are installing for all users or just you. If you answer 'Just you' then no admin privileges are required. No need to talk to IT departments (always a good thing).

**Reply by edvlmaas** [reply]
In my case, I still had to go through IT to install it on my D: drive, but we installed it just for me (not all users) and it worked. Thanks so much!

**Comment by edvlmaas** [comment]
Thank you, Dean. I will give this a try!
